src/core/component.test.ts

```diff
@@ -1178,6 +1186,53 @@ describe("template", () => {
         ).toBeUndefined();
     });
 
+    it("should wrap hidden elenents", (done) => {
```

Comment:
Is there a test that checks the case when no content rendered? E.g. `showTemplate === false`, `innerHTML` is empty

Reply:
I added it in the test